README.md

```diff
@@ -219,6 +220,19 @@ await client.logon();
 await client.logoff();
 ```
 
+## onSessionExpiration callback
+The onSessionExpiration is callback with the sdk client as parameter, it is called when the ACC session is expired.
+The callback return no thing. We can use the callback to reconnect the sdk client.
```

Comment:
The callback return **nothing**. We can use the callback to reconnect the sdk client.